src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,120 @@
 package org.example;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
 public class Main {
+
+    static List<String> options = new ArrayList<>(List.of("add", "remove", "list", "exit"));
+    static List<List<String>> tasks = new ArrayList<>();
+    static Scanner scanner = new Scanner(System.in);
+
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        readTasks("tasks.csv");
+        selectOption();
+    }
+
+    public static void readTasks(String fileName) {
+        Path path = Paths.get(fileName);
+        try {
+            List<String> lines = Files.readAllLines(path);
+            for (String line : lines) {
+                String[] words = line.split(",");
+                tasks.add(new ArrayList<>(List.of(words)));
+            }
+        } catch (IOException e) {
+            System.out.println("File not found.");
```

Comment:
BUG: IOException nemusi byt vzdy "file not found" prepsal bych to, plus chybove zpravy logovat do err konzole. Taky je dobre dat info ze aplikace dal bude pokracovat s prazdnym listem ukolu.

```suggestion
            System.err.println("Failed to load tasks.csv: " + e.getMessage());
            System.err.println("Application will work with empty tasks list");
```